pages/about.html

```diff
@@ -1,12 +1,173 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About author</title>
+  <link href="../styles/style.css" rel="stylesheet" />
+  <link href="../styles/about.css" rel="stylesheet" />
+
 </head>
+
 <body>
-  
+
+  <div class="content-about-me">
+
+    <nav class="nav-container">
+
+      <ul>
+        <li class="nav-item active">
+          <a href="index.html">Home</a>
+        </li>
+        <li>
+          <a href="about.html" class="active">About me</a>
+        </li>
+        <li>
+          <a href="portfolio.html">Portfolio</a>
+        </li>
+
+      </ul>
+
+    </nav>
+
+
+    <div class="content-about-me">
+
+      <div class="about-me-header">
+        <h2>Olena Rostotskyy</h2>
+        <h3>Junior software developer</h3>
```

Comment:
You skipped `h1` here but all of your other heading tags look good.  

Remember, if `h1` is too big you can always make it smaller with CSS.